### ISSUE TITLE
Pedia should have a more easily found article explaining planet's unit health in combat

### ISSUE THREAD

**Issue by Oberlus** [freeorion/freeorion#2315]
### Environment
* **FreeOrion Version**: 2018-11-04.25ae12b
* **Operating System**: Ubuntu 16.04
  * Compiled from source

### Description
![wrongcombatreportplanetshp](https://user-images.githubusercontent.com/28650914/48126339-26897000-e281-11e8-8dbd-c750cda79001.png)
The combat report shows Unit Health 9.0/22.0 for a planet that had initially 1 shield and 5 planetary defense (maybe 6 at the start of combat? it was set to defense focus on the turn before combat ensued) and received during the combat two shots of 5 damage.
The planet meters in the map show correctly that the planet has now 0 shield and 0 planetary defense.
Also, the combat report log doesn't say that the planet was incapacitated, when IIRC this was indicated in previous versions of FO.

### Expected Result
I guess the combat report should show Unit Health 0.0/6.0, if meter's initial values were to be considered, or 0.0/11.0, if meter's max value were the right ones, but certainly not 0.0/22.0 and less a positive value in the remaining unit health. 

### Steps to reproduce
Load attached save game and press next turn twice. Combat should ensue at Veza system.
Check combat report summary and log.

[FO_IncorrectCombatReport_0048_2018-11-07.sav.zip](https://github.com/freeorion/freeorion/files/2557031/FO_IncorrectCombatReport_0048_2018-11-07.sav.zip)

**Comment by Dilvish-fo** [comment]
Planet combat 'health' is the sum of shields + defense + infrastructure.  As your screenshot shows, this planet still has 9 infrastructure, hence the 9/22 (without more meter info I couldn't say exactly how the baseline added up to 22, but I expect it is correct).

Trying to check what explanation for this we have in the Pedia, I did not immediately finding any.  It seems odd to me that we do not have a "Combat" entry under the "Game Concepts" section.  I had to check the entry for Shield parts and follow a link there to "Damage" to get this info about planet health.

Sure our combat is fairly typical in a general sense, but clearly there are details like this that need to be more clearly explained.  I think we should have a "Combat" article under "Game Concepts".
